src/backend/web.rs

```diff
@@ -23,14 +24,26 @@ use wasm_bindgen::prelude::*;
 // type is (for now) harmless.  Eventually wasm32 will support threading, and depending on how this
 // is integrated (or not integrated) with values like those in webgpu, this may become unsound.
 
-#[derive(Debug, Clone)]
+//forse run from integration system
+
+#[derive(Clone, Debug)]
 pub(crate) struct Sendable<T>(T);
 unsafe impl<T> Send for Sendable<T> {}
 unsafe impl<T> Sync for Sendable<T> {}
 
-pub(crate) type Context = Sendable<web_sys::Gpu>;
+pub(crate) struct Context(web_sys::Gpu);
+unsafe impl Send for Context {}
+unsafe impl Sync for Context {}
```

Comment:
Is this necessary?

Reply:
Good question! It was introduced in #272, which appears to be focused on associated types being Send/Sync. Maybe the context doesn't need it? cc @kyren 
Let's remove this Send/Sync from the Context for now and see?

Reply:
It might not be necessary since `Context` is strictly an internal type?  If it's not visible at all to the user and anything that contains `Context` is itself forcibly made `Send + Sync` then I guess It shouldn't matter?  I don't remember specifically but I probably made `Context` Send just to follow the pattern of forcibly making the all the web_sys handle types `Send + Sync` and focusing my effort from the bottom up, but I'd 100% believe that it might not have been actually necessary.